b2sdk/sync/folder.py

```diff
@@ -258,16 +254,18 @@ def _walk_relative_paths(self, local_dir, b2_dir, reporter, policies_manager):
                     file_mod_time = get_file_mtime(local_path)
                     file_size = os.path.getsize(local_path)
 
-                    # if policies_manager.should_exclude_file_version(version):  TODO: fix method name
-                    #     continue
-
-                    yield LocalSyncPath(
+                    local_sync_path = LocalSyncPath(
                         absolute_path=self.make_full_path(b2_path),
```

Comment:
this is really funny - we make `absolute_path` (local) from `b2_path` - as I mentioned it should be `relative_path`